packages/dashboard/e2e/tests/utils/getDecimalPlaces.ts

```diff
@@ -0,0 +1,4 @@
+export const getDecimalPlaces = (num: number | string | null) => {
```

Comment:
`Uncaught TypeError: Cannot read properties of undefined (reading 'replace')` on integers

Reply:
```
> getDecimalPlaces(3)
Uncaught TypeError: Cannot read properties of undefined (reading 'replace')
    at dd (REPL8:3:38)

> dd("3")
Uncaught TypeError: Cannot read properties of undefined (reading 'replace')
    at dd (REPL8:3:38)
```

Reply:
resolved :) now works with integer and double values